test/time_helpers.js

```diff
@@ -0,0 +1,15 @@
+contract('TimeHelpers test', accounts => {
+  let timeHelpersMock
+
+  before(async () => {
+    timeHelpersMock = await artifacts.require('TimeHelpersMock').new()
+  })
+
+  it('checks block number', async () => {
+    assert.equal((await timeHelpersMock.getBlockNumberExt.call()).toString(), (await timeHelpersMock.getBlockNumber64Ext.call()).toString(), "block number should match")
```

Comment:
We should also test that the returned value is the actual timestamp or block number.